docs/source/distributions/configuration.md

```diff
@@ -385,6 +385,121 @@ And must respond with:
 
 If no access attributes are returned, the token is used as a namespace.
 
+### Access control
+
+When authentication is enabled, access to resources is controlled
+through the `access_policy` attribute of the auth config section under
+server. The value for this is a list of access rules.
+
+Each access rule defines a list of actions either to permit or to
+forbid. It may specify a principal or a resource that must match for
+the rule to take effect.
+
+Valid actions are create, read, and delete. The resource to match
+should be specified in the form of a type qualified identifier, e.g.
+model::my-model or vector_db::some-db, or a wildcard for all resources
+of a type, e.g. model::*. If the principal or resource are not
+specified, they will match all requests.
+
+A rule may also specify a condition, either a 'when' or an 'unless',
+with additional constraints as to where the rule applies. The
+constraints supported at present are:
+
+ - 'user with <attr-value> in <attr-name>'
+ - 'user with <attr-value> not in <attr-name>'
+ - 'user is owner'
+ - 'user is not owner'
+ - 'user in owners <attr-name>'
+ - 'user not in owners <attr-name>'
+
+The attributes defined for a suer will depend on how the auth
```

Comment:
```suggestion
The attributes defined for a user will depend on how the auth
```

Reply:
Oops! Sorry! Fixed now, thanks for spotting that!